memtest/build.gradle

```diff
@@ -32,7 +34,7 @@ kotlin {
 
 task runProgram {
     def buildType = 'DEBUG' // Change to 'DEBUG' to run application with debug symbols.
-    dependsOn kotlin.targets.macos.compilations.main.linkTaskName('EXECUTABLE', buildType)
+    dependsOn kotlin.targets.macos.binaries.getExecutable("DEBUG").linkTask
```

Comment:
What's going on here? This looks like it will change behavior.

Reply:
no this is just a change in the dsl for how you get the main executable's linkTask, but I should be using the buildType var. Will fix that.